### ISSUE TITLE
Control panel - Website logs

### ISSUE THREAD

**Issue by ddrury** [fisharebest/webtrees#2644]
On clicking the search button am immediately returned to "My page"

**Comment by fisharebest** [comment]
This seems to be working in the latest code.  Can you confirm?

**Comment by ddrury** [comment]
No, still going straight to my page. Looking at the console I see the following 3 urls when I click the search. The 2nd one looks suspect doesn't it?
```
http://webtrees.home.lan/index.php?route=http%3A%2F%2Fwebtrees.home.lan%2Findex.php%3Froute%3D%252Fadmin%252Flogs&action=show&from=2019-09-30&to=2019-10-20&type=&ip=&text=&username=&gedc=
http://webtrees.home.lan/index.php?route=%2F
http://webtrees.home.lan/index.php?route=%2Ftree%2Fdrury%2Fmy-page
```

**Comment by ddrury** [comment]
BTW last commit was https://github.com/fisharebest/webtrees/issues/2648

**Comment by bmhm** [comment]
Yes this is still an issue. If you use only 

```
    <input type="hidden" name="route" value="/admin/logs">
```
It will work.

`e(route('action-name'))` will somehow insert the full URL.

**Comment by bmhm** [comment]
@ddrury probably fixed by greg in 54a7c5882.
Can you try again?

**Comment by ddrury** [comment]
The problem is that in AdminSiteController::logs(), QueryParams[] in $server is empty, 'action' is the one needed for the JS to kick in and force datatables to update.
